atlasdb-ete-tests/docker/conf/atlasdb-ete.cassandra-async.yml

```diff
@@ -41,3 +41,10 @@ atlasdb:
       keyStorePath: "var/security/keyStore.jks"
       keyStorePassword: "keystore"
       trustStorePath: "var/security/trustStore.jks"
+
+atlasDbRuntime:
```

Comment:
As mentioned in the PR description, only moving a few ETEs over to have a spread of some install defined stuff and some runtime defined etes